pkg/orchestrator/scanestimationwatcher/watcher.go

```diff
@@ -512,6 +515,23 @@ func (w *Watcher) reconcileInProgress(ctx context.Context, scanEstimation *model
 	return nil
 }
 
+func updateTotalScanTimeWithParallelScans(scanEstimation *models.ScanEstimation) {
+	if scanEstimation == nil || scanEstimation.ScanTemplate == nil {
+		return
+	}
+
+	maxParallelScanners := utils.ValueOrZero(scanEstimation.ScanTemplate.MaxParallelScanners)
+
+	if maxParallelScanners > 1 {
+		numberOfJobs := *scanEstimation.Summary.JobsCompleted
```

Comment:
```suggestion
                var numberOfJobs int
                
                if scanEstimation.Summary != nil {
                    numberOfJobs = utils.ValueOrZero(scanEstimation.Summary.JobsCompleted)
                }
```

or 

```suggestion
                if scanEstimation.Summary == nil {
                    return
                }
                
                numberOfJobs := utils.ValueOrZero(scanEstimation.Summary.JobsCompleted)
```

Reply:
I was not checking this field for nil since this is an internal function that being called once, and in the context it is being called, scanEstimation.Summary is already set and cannot be nil. 
Do you think we still need to check for nil in that case?